### ISSUE TITLE
Set Azure DataBase for Receiver Ingest Messages

### ISSUE THREAD

**Issue by cosmycx** [CDCgov/data-exchange-hl7#34]
This is needed for the Receiver Re-play message (file) functionality. 
- [ ] Design Solution - Choose DB to use
- [ ] Integrate to Azure Vault

**Comment by RuDue** [comment]
@lmcnabb How many steps in the deduplication process?   Would this need to be pulled out into it's own epic?  See #202 vs. #86

**Comment by lmcnabb** [comment]
Yes, I think DeDuper is a separate epic.  We haven't discussed/planned how we are going to dedupe yet.
________________________________
